gradle.properties

```diff
@@ -17,9 +17,13 @@ POM_DEVELOPER_NAME=Mapbox
 VERSION_NAME=1.1.1-SNAPSHOT
 POM_ARTIFACT_ID=mapbox-android-sdk-versions
 POM_NAME=Mapbox Android SDK Versions
+POM_ARTIFACT_TITLE=Mapbox Android SDK Version Plugin
 POM_DESCRIPTION=Mapbox Android SDK Version persistor
 POM_PACKAGING=jar
 
+SDK_REGISTRY_NAME=mapbox-android-sdk-versions
```

Comment:
Maybe mapbox-android-sdk-versions-plugin? Or android-sdk-versions-plugin if we want to follow SDK_REGISTRY_NAME == `repo-name` pattern?

Reply:
Agree, lets use the repo name.